readme.md

```diff
@@ -67,6 +67,7 @@ The community is encouraged to port Rosé Pine to their favourite apps 🌸
 - [Matplotlib](https://github.com/h4pZ/rose-pine-matplotlib) by [h4pZ](https://github.com/h4pz)
 - [Typora](https://github.com/rose-pine/typora)
 - [Zathura](https://github.com/edunfelt/zathura) by [edunfelt](https://github.com/edunfelt)
+- [Obsidian](https://github.com/mimishahzad/rose-pine-moon-obsidian) 
```

Comment:
Could we move this above Typora to keep it alphabetical

Reply:
Also, please credit yourself, eg. add `by [you](your-github-url or similar)` to the end